README.md

```diff
@@ -91,8 +91,9 @@ Alternatively, you can use [Homebrew](http://brew.sh/) or
 ```sh
 brew install fzf
 
-# Install shell extensions
-/usr/local/opt/fzf/install
+# To install useful keybindings and fuzzy completion:
```

Comment:
Nit: `keybindings` -> `key bindings`

Reply:
I took it directly from the command line output. Hence, the output of the command line needs an update, too?

Reply:
Yes, I'd like that, but we can get it right here first.